.github/workflows/maven-release.yml

```
@@ -105,6 +105,7 @@ jobs:
         run: |
           mvn -V -B -e versions:set -DnewVersion=${{ env.RELEASE_VERSION }} -DprocessAllModules -DgenerateBackupPoms=false
           mvn -V -B -e versions:set-scm-tag -DnewTag=${{ env.RELEASE_VERSION }} -DgenerateBackupPoms=false
+          mvn -V -B -e clean formatter:format sortpom:sort impsort:sort -Pautoformat
```

```suggestion
          mvn -V -B -e clean sortpom:sort -Pautoformat
```
Maybe just limit this to only sorting the pom?